### DISCUSSION TITLE
Accessibility service stops

### DISCUSSION THREAD

**Discussion by mgndnr1** [d4rken-org/sdmaid-se#975] [Q&A] [answered]
Although I give all permissions needed accessibility service regularly stops and I restart the service or even the phone. It works for some time but then accessibility service stops again. Android 13, Xiaomi Redmi Note 10 Pro, No root, SD Maid SE Foss (0.19.1 beta)

**Comment by d4rken** [comment, marked as answer]
This is a ROM issue with many Chinese phones, often with MIUI, see here for tipps and tricks:
https://github.com/d4rken-org/sdmaid-se/wiki/Setup#acs-troubleshooting
